spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -573,6 +573,44 @@ func TestCreateDatabaseWithDefaultLeaderSample(t *testing.T) {
 	assertContains(t, out, "The result of the query to get")
 }
 
+func TestCustomInstanceConfigSample(t *testing.T) {
+	_ = testutil.SystemTest(t)
+	t.Parallel()
+
+	projectID := getSampleProjectId(t)
+	defer cleanupInstanceConfigs(projectID)
+
+	var b bytes.Buffer
+	baseConfig := fmt.Sprintf("projects/%s/instanceConfigs/%s", projectID, "nam11")
+	userConfig := fmt.Sprintf("custom-golang-samples-config-%v", randomID())
+	if err := createInstanceConfig(&b, "projects/"+projectID, userConfig, baseConfig); err != nil {
+		t.Errorf("failed to create instance configuration: %v", err)
+	}
+	out := b.String()
+	assertContains(t, out, "Created instance configuration")
+
+	b.Reset()
+	if err := updateInstanceConfig(&b, fmt.Sprintf("projects/%s/instanceConfigs/%s", projectID, userConfig)); err != nil {
+		t.Errorf("failed to update instance configuration: %v", err)
+	}
+	out = b.String()
+	assertContains(t, out, "Updated instance configuration")
+
+	b.Reset()
+	if err := listInstanceConfigOperations(&b, fmt.Sprintf("projects/%s", projectID)); err != nil {
+		t.Errorf("failed to list instance configuration operations: %v", err)
+	}
+	out = b.String()
+	assertContains(t, out, "List instance config operations")
+
+	b.Reset()
+	if err := deleteInstanceConfig(&b, fmt.Sprintf("projects/%s/instanceConfigs/%s", projectID, userConfig)); err != nil {
+		t.Errorf("failed to delete instance configuration: %v", err)
+	}
+	out = b.String()
+	assertContains(t, out, "successfully deleted")
```

Comment:
```suggestion
	assertContains(t, out, "Deleted instance configuration")
```
Added suggestion to change the `Fprintf` statement inside corresponding sample.